src/Form/Type/UiElement/LogosUiElementType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz's SyliusUiElementsPlugin for Sylius.
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusUiElementsPlugin\Form\Type\UiElement;
+
+use MonsieurBiz\SyliusRichEditorPlugin\Attribute\AsUiElement;
+use MonsieurBiz\SyliusRichEditorPlugin\Attribute\TemplatesUiElement;
+use MonsieurBiz\SyliusUiElementsPlugin\Form\Type\ImageType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+#[AsUiElement(
+    code: 'monsieurbiz_ui_elements.logos_ui_element',
+    icon: 'images',
+    title: 'monsieurbiz_ui_elements.ui_element.logos_ui_element.title',
+    description: 'monsieurbiz_ui_elements.ui_element.logos_ui_element.description',
+    templates: new TemplatesUiElement(
+        adminRender: '@MonsieurBizSyliusUiElementsPlugin/Admin/UiElement/logos_ui_element.html.twig',
+        frontRender: '@MonsieurBizSyliusUiElementsPlugin/Front/UiElement/logos_ui_element.html.twig',
+    ),
+    tags: [],
+)]
+class LogosUiElementType extends AbstractType
+{
+    /**
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('title', TextType::class, [
+                'label' => 'monsieurbiz_ui_elements.common.fields.title',
+                'required' => false,
+            ])
+            ->add('logos', CollectionType::class, [
+                'label' => 'monsieurbiz_ui_elements.common.fields.logos',
+                'entry_type' => ImageType::class,
+                'entry_options' => [
+                    'with_link' => false,
+                    'with_alignment' => false,
+                ],
+                'allow_add' => true,
+                'allow_delete' => true,
+                'constraints' => [new Assert\Valid()],
+                'attr' => [
+                    'class' => 'ui segment',
```

Comment:
```suggestion
                    'class' => 'ui segment secondary collection--flex',
```

Can you add `collection--flex` class ? And `secondary` to have grey background.
The class `collection--flex` is not used in the test app, but in pagaio it will be displayed like this : 

![image](https://github.com/user-attachments/assets/002e4213-8d82-441d-addb-406815fd4f84)

https://github.com/monsieurbiz/SyliusCaseStudyPlugin/blob/master/src/Form/Type/UiElement/CaseStudyUiElementType.php#L56